google-cloud-language/lib/google/cloud/language.rb

```diff
@@ -54,10 +54,10 @@ module Cloud
     # ## Creating documents
     #
     # Cloud Natural Language API supports UTF-8, UTF-16, and UTF-32 encodings.
```

Comment:
This has been changed.

@landrito, @quartzmo and I paired on this and revised all the documentation. We removed mention of encoding from everywhere except for the `offset` values. Can you take a look and tell us if this new verbiage works for you.